app/views/global_error.scala.html

```diff
@@ -33,7 +35,8 @@ <h1 id="header" class="column-two-thirds">@messages(heading)</h1>
 }
 
 @mainContent = {
- <p class="column-two-thirds">@messages(message)<a href=@messages(hrefLink) target="_blank">@messages(hrefMessages)</a>@messages(postHref)</p>
+ <p class="column-two-thirds" id="message1">@messages(message)<a href=@messages(hrefLink) target="_blank">@messages(hrefMessages)</a>@messages(postHref)</p>
+ <p class="column-two-thirds" id="message2">@messages(secondParagraph)<a href=@messages(hrefLink) target="_blank">@messages(hrefMessages)</a>@messages(postHref)</p>
```

Comment:
My gut instinct is that we probably do not need the hrefLink, hrefMessages and postHref references as they are present above so if both messages are displayed we could end up with these href messages being rendered twice?